apps/bfd-data-npi/pom.xml

```diff
@@ -0,0 +1,155 @@
+<project
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+		<parent>
+		<groupId>gov.cms.bfd</groupId>
+		<artifactId>bfd-parent</artifactId>
+		<version>1.0.0-SNAPSHOT</version>
+	</parent>
+
+	<groupId>gov.cms.bfd.data.npi</groupId>
+	<artifactId>bfd-data-npi</artifactId>
+	<packaging>jar</packaging>
+	<version>1.0-SNAPSHOT</version>
+
+	<description>
+		NPI Org code lookup jar
+	</description>
+
+	<properties>
+		<junit>3.8.1</junit>
+	</properties>
```

Comment:
why is this on an old version of junit? can't we just use the latest via inheritance from the parent pom?